Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -0,0 +1,93 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
+import UIKit
+
+internal final class AddNewCardViewController: UIViewController {
+  @IBOutlet fileprivate weak var cardNumberLabel: UILabel!
+  @IBOutlet fileprivate weak var cardNumberTextField: UITextField!
+  @IBOutlet fileprivate weak var cardholderNameLabel: UILabel!
+  @IBOutlet fileprivate weak var cardholderNameTextField: UITextField!
+  @IBOutlet fileprivate weak var expirationLabel: UILabel!
```

Comment:
I know it's not part of the scope, but i wonder if it would be a good idea to limit the number of characters in `5` or `7` for this textfield and create a mask for it. So after adding the first 2 digits, let's say `11`, the textField would automatically append a `\` character, and adding the year `23` (or `2023`), the textfield would show `11/23` or `11/2023`. We can't talk about it with @dannyalright and I can create an UITextField extension to handle that.